CHANGELOG.md

```diff
@@ -55,12 +55,20 @@
   * Amazon 2 x86_64 binaries
     * 2.5.8 2.5.9 2.6.10 2.6.8 2.6.9 2.7.3 2.7.4 2.7.5 2.7.6 2.7.7 3.0.0 3.0.3 3.0.4 3.0.5 3.1.2 3.1.3 3.2.0 3.2.1 [\#5314](https://github.com/rvm/rvm/pull/5314)
 * Debian Family
+  * Debian 11 x86_64 binaries
+    * 2.5.9 2.6.10 2.7.0 2.7.1 2.7.2 2.7.3 2.7.4 2.7.5 2.7.6 2.7.7
+    * 3.0.0 3.0.1 3.0.2 3.0.3 3.0.4 3.0.5 3.1.0 3.1.1 3.1.2 3.1.3 3.2.0 3.2.1
   * Debian 10 x86_64 binaries
-    * 2.5.9 2.6.10 2.6.7 2.6.8 2.6.9 2.7.3 2.7.4 2.7.6 2.7.7 3.0.0 3.0.2 3.0.4 3.0.5 3.1.2 3.1.3 3.2.0 3.2.1 [\#5314](https://github.com/rvm/rvm/pull/5314)
```

Comment:
Do not remove references to PR. They are helpful to figure out when things changes if something breaks.

Reply:
but they are part of the git history... I figured with binaries, they don't add much values. the same PR number are shown on many lines.. which isn't as accurate as the git history actually..
so that's why I thought, lets just remove those...

Reply:
Its way easier to get to them from the changeling though, so please keep it.